README.python.md

```diff
@@ -17,3 +17,64 @@ We currently ship binary wheels on pypi for:
 
 If you need another platform you will have to build from source using the repository at https://github.com/n0-computer/iroh-ffi/
 
+
+## Development setup
```

Comment:
lets put this into `DEVELOPERS.md` please, as this readme gets published to pypy

Reply:
Aha!
It was in the main README.md before, can also move back there?
Should the readme include a short example then maybe?

Reply:
moved things around. wasn't super sure what should go where